api/proto/controller/v1/controller_service.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+package controller.v1;
+
+import "common/v2/common_v2.proto";
+
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/controller/v1";
+
+// ControllerService defines the APIs for the controller.
+//
+// Currently, this API is only intended for *internal* consumption: this is a way for different parts of the disperser
+// to communicate with each other
+service ControllerService {
+  // AuthorizePayment handles payment authorization for blob dispersal
+  //
+  // This is intended to be called by API server instances that are handling dispersal requests. The controller
+  // is responsible for accounting and metering for the dispersal.
+  rpc AuthorizePayment(AuthorizePaymentRequest) returns (AuthorizePaymentResponse) {}
+}
+
+// Contains all information necessary for the controller to evaluate the validity of a dispersal payment
+message AuthorizePaymentRequest {
+  // The blob header is used for the following purposes:
+  // 1. Contains the PaymentHeader, which describes the payment being offered
+  // 2. Contains the quorums being dispersed to
+  common.v2.BlobHeader blob_header = 1;
+
+  // Signature over the keccak hash of this AuthorizePaymentRequest (excluding this signature field).
+  bytes disperser_signature = 2;
+}
+
+// AuthorizePaymentResponse is returned after the controller does accounting and metering.
+//
+// A GRPC error indicates that there was a problem with the payment authorization. No error means everything succeeded.
+//
+// TODO(litt3): We should document the potential errors that could be returned.
```

Comment:
In general, we should be careful about putting TODOs in protobuf docs, since this is public documentation. Perhaps add the TODO as a stand alone comment that isn't part of the message doc?